compass_app/app/test/data/repositories/activity/activity_repository_remote_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:compass_app/data/repositories/activity/activity_repository.dart';
+import 'package:compass_app/data/repositories/activity/activity_repository_remote.dart';
+import 'package:compass_app/utils/result.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../../../testing/fakes/services/fake_api_client.dart';
```

Comment:
Why do you have both `test` and `testing`? What problem does this organisation fix?

Reply:
This came up during review. Testing is for shared test enablement code like common mocks or default complex data objects. 
Test is for tests that run.

Reply:
I would think of testing like production code that does not ship to customers of your package.